controllers/users.js

```diff
@@ -28,9 +27,9 @@ router.post('/', function(req, res) {
 // Update a user
```

Comment:
ya, the client should already know what endpoint they are hitting, so they know what schema to expect coming back